### PR TITLE
Launch Script

### PR DESCRIPTION
**Pull request by enjeyw**
Scripting to help us consistently set up GE platform

### CODE REVIEW

**Comment by tristanhcole** [comment]
https://giphy.com/gifs/excited-screaming-jonah-hill-5GoVLqeAOo6PK

File: config.py [hunk 1]
```diff
@@ -268,7 +268,6 @@ def get_database_uri(name, host, censored=True):
 RESERVE_TOKEN_ADDRESS = specific_parser['ETHEREUM'].get('reserve_token_address')
 RESERVE_TOKEN_NAME = specific_parser['ETHEREUM'].get('reserve_token_name')
 RESERVE_TOKEN_SYMBOL = specific_parser['ETHEREUM'].get('reserve_token_symbol')
-# RESERVE_TOKEN_DECIMALS = int(specific_parser['ETHEREUM'].get('reserve_token_decimals', 18))
```
Comment (tristanhcole) [inline config.py]: @enjeyw Accidental?

File: app/server/api/organisation_api.py [hunk 2]
```diff
@@ -54,33 +55,45 @@ def get(self, organisation_id):
...
+            new_organisation.bind_token(token)
+
+        elif deploy_cic:
+
+            cic_response_object, cic_response_code = deploy_cic_token(post_data, new_organisation)
+            if cic_response_code == 201:
+                response_object['data']['token_id'] = cic_response_object['data']['token_id']
+            else:
+                return make_response(jsonify(cic_response_object)), cic_response_code
+
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/api/organisation_api.py]: only if u decided to - you can also create an organisation and assign it to an existing token, which is what you do with a reserve currency (which is to say `deploy_cic_token` isn't used for reserves)

File: app/server/models/organisation.py [hunk 3]
```diff
@@ -38,6 +44,13 @@ class Organisation(ModelBase):
         primaryjoin="Organisation.org_level_transfer_account_id==TransferAccount.id",
         uselist=False)
 
+    # TODO: This is a hack to get around the fact that org level TAs don't always show up. Super not ideal
+    @property
+    def queried_org_level_transfer_account(self):
+        if self.org_level_transfer_account_id:
+            return server.models.transfer_account.TransferAccount.query.get(self.org_level_transfer_account_id)
+        return None
+
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/models/organisation.py]: yeah it's a fix for `org_level_transfer_account` just randomly not showing up. I hate it, but unless we work out what's going on with SQLAlchemy, we don't really have a choice.

File: app/server/api/transfer_account_api.py [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
     view_transfer_account_schema, view_transfer_accounts_schema
-from server.utils.auth import requires_auth
+from server.utils.auth import requires_auth, show_all
 from server.utils.access_control import AccessControl
 
 transfer_account_blueprint = Blueprint('transfer_account', __name__)
 
 
 class TransferAccountAPI(MethodView):
+    # @show_all
```
[the remark this answers is not in the document]
Comment (enjeyw) [reply app/server/api/transfer_account_api.py]: ohhh  yeah this was me debugging the big baked query import. will remove

File: config.py (hunk 1 above)
Comment (enjeyw) [reply config.py]: nah all good - didn't wanna delete, so purposely commented out.

File: app/server/api/organisation_api.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (tristanhcole) [reply app/server/api/organisation_api.py]: ok that makes sense

**Pull request merged**